cea/globalvar.py

```diff
@@ -77,7 +77,7 @@ def __init__(self):
         # Commands for the evolutionary algorithm
 
 
-        self.initialInd = 2  # number of initial individuals
+        self.initialInd = 5  # number of initial individuals
```

Comment:
yep, on it. Moving these to the optimization_settings.py file in another branch. Will soon send it for PR :)